src/Map/ProspectInfo.cs

```diff
@@ -64,6 +64,24 @@ internal class ProspectInfo
         public readonly int X;
         public readonly int Z;
 
+        /// <summary>
+        /// We don't want to change the serialization format,
+        /// so we keep the coordinates in a cache.
+        /// </summary>
+        [Newtonsoft.Json.JsonIgnore]
+        private ChunkCoordinates? _coordinates = null;
+        [Newtonsoft.Json.JsonIgnore]
+        public ChunkCoordinates ChunkCoordinates
```

Comment:
i would prefer the class to be called coordinate instead of plural (just a small nitpick)